clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -2077,6 +2077,21 @@ void Sema::finalizeSYCLDelayedAnalysis(const FunctionDecl *Caller,
   }
 }
 
+bool Sema::isArrayZeroInitialized(VarDecl *VD, bool CheckValueDependent) {
```

Comment:
what is the purpose of CheckValueDependent here?  It doesn't seem to be used.  Additionally, you likely don't need to use it up the call chain either.